labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,124 @@
+var playerHP = 40;
+var grantHP = 10;
+var playersName = "";
+var grantDeaths = 0;
+
+function startGame() {
+	 wouldYou = prompt("Would you like to play?");
+
+	if (wouldYou === "yes") {
+		playersName = prompt("What would you like to name your hero?");
+		
+		startCombat();
+	}
+}
+
+function startCombat() {
+	playerHP = 40;
+	grantDeaths = 0;
+	while (grantDeaths <= 3 && playerHP >= 0) {
+		 battle();
+	}
+	if (grantDeaths === 3) {
+	declareWinner();
+
+   }else if (playerHP <=0) {
+	declareWinner();
+	}
+
+}
+
+function battle() {
+	grantHP = 10;
+	while (grantHP >= 0 && playerHP >= 0) {
+		round();
+	}
+ 	if (grantHP <= 0) {
+	grantDeaths = grantDeaths + 1;
+	console.log(playersName + ' has won a round');
+	}
+
+}
+
+function round() {
+	
+	attack = prompt('Do you want to attack? or quit?')
```

Comment:
The instructions want you to ask me if I want to attack before each round of dealing damage, not before each round of killing.